sanic/app.py

```diff
@@ -545,7 +545,7 @@ def test_client(self):
     def run(self, host=None, port=None, debug=False, ssl=None,
             sock=None, workers=1, protocol=None,
             backlog=100, stop_event=None, register_sys_signals=True,
-            log_config=LOGGING):
+            log_config=None):
```

Comment:
Why is this needed? This changes the current behavior and no longer use the `LOGGING` config from sanic/config.py by default.